sdk/src/client/secret/ledger_nano.rs

```diff
@@ -306,8 +306,10 @@ impl SecretManage for LedgerSecretManager {
         } else {
             // figure out the remainder output and bip32 index (if there is one)
             #[allow(clippy::option_if_let_else)]
-            let (remainder_output, remainder_bip32) = match &prepared_transaction.remainder {
-                Some(remainder) => {
+            let (remainder_output, remainder_bip32) = match &prepared_transaction.remainders.as_deref() {
+                // Multiple remainder outputs would require blind signing since at least one would contain a native
+                // token, that's why matching a single one is enough.
+                Some([remainder]) => {
```

Comment:
This will only match if the length of the slice is 1, I believe. You can do this instead I think
```suggestion
                Some([remainder, ..]) => {
```

Reply:
But I want it only for the length of 1

Reply:
ohhh, I misunderstood